tls/tls-full/docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+version: '3.5'
+
+services:
+  cassandra:
+    image: cassandra:3.11
+    ports:
+      - "9042:9042"
+  temporal:
+    image: temporalio/auto-setup
+    ports:
+      - "7233:7233"
+    volumes:
+      - ${DYNAMIC_CONFIG_DIR:-../config/dynamicconfig}:/etc/temporal/config/dynamicconfig
+      - ${TEMPORAL_LOCAL_CERT_DIR}:${TEMPORAL_TLS_CERTS_DIR}
+      - ./config_template.yaml:/etc/temporal/config/config_template.yaml
+    environment:
+      - "CASSANDRA_SEEDS=cassandra"
+      - "DYNAMIC_CONFIG_FILE_PATH=config/dynamicconfig/development.yaml"
+      - "SKIP_DEFAULT_NAMESPACE_CREATION=true"
+    depends_on:
+      - cassandra
+  temporal-admin-tools:
+    image: temporalio/admin-tools
+    stdin_open: true
+    tty: true
+    volumes:
+      - ${TEMPORAL_LOCAL_CERT_DIR}:${TEMPORAL_TLS_CERTS_DIR}
+    environment:
+      - "TEMPORAL_CLI_ADDRESS=temporal:7233"
+      - "TEMPORAL_CLI_TLS_CA=${TEMPORAL_TLS_CERTS_DIR}/cluster/ca/server-intermediate-ca-chain.pem"
+      - "TEMPORAL_CLI_TLS_CERT=${TEMPORAL_TLS_CERTS_DIR}/cluster/internode/cluster-internode.pem"
+      - "TEMPORAL_CLI_TLS_KEY=${TEMPORAL_TLS_CERTS_DIR}/cluster/internode/cluster-internode.key"
+      - "TEMPORAL_CLI_TLS_ENABLE_HOST_VERIFICATION=true"
```

Comment:
for the sample to work, then user must remember to update their /etc/hosts file right? Is that part of this sample or at least documented somewhere?